CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- Add cumulative gauge API
```

Comment:
I think it's called "cumulative API" instead of "cumulative gauge".

Reply:
Another adjective promoted to nounhood. Changed.